src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java

```diff
@@ -235,127 +301,115 @@ public void testWrapAssertionErrorWithoutMessage()
         }
     }
 
+    @Test()
+    public void testIsThrowableCausedBy()
+    {
+        Throwable causedByIOException = new RuntimeException("This is runtime exception", new AssertionError("this is assertion error", new IOException("this is final cause")));
+        Assert.assertTrue("Throwable has unexpected cause", SelenideAddons.isThrowableCausedBy(causedByIOException, IOException.class));
+    }
+
+    @Test()
+    public void testIsThrowableCausedByMessage()
```

Comment:
add `@SuppressBrowsers` since a browser is not needed for this unit test

Reply:
Removed